pkg/image/image-list.go

```diff
@@ -0,0 +1,5 @@
+package images
+
+var Images = []string{
+	"docker.io/envoyproxy/envoy:v1.30.1",
```

Comment:
You need to list the actual images, this will require to modify two places next time we update the existing image

Reply:
@aojea You’re absolutely right. I’ve updated the code to centralize the image definitions as suggested. If you have any other feedback, please let me know!